interface/weno/RxLog.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
Need license/author header and need to place the 2 security flags prior to the globals.php include.